io/net/virt-net/network_virtualization.py

```diff
@@ -153,7 +153,7 @@ def setUp(self):
                'failover' in str(self.name.name):
                 self.cancel("this test is not needed")
         self.local = LocalHost()
-        cmd = "echo 'module ibmvnic +pt; func send_subcrq -pt' > /sys/kernel/debug/dynamic_debug/control"
+        cmd = "echo 'module ibmvnic +p; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control"
```

Comment:
why remove timestamp ? it was suggested from developer to add `+pt` 
`echo 'module ibmvnic +pt; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control`

Reply:
+1, please do not remove the +pt arg.